client/my-sites/themes/thanks-modal.jsx

```diff
@@ -201,20 +200,15 @@ const ThanksModal = React.createClass( {
 
 export default connect(
 	( state, { site } ) => {
-		if ( ! site ) {
-			return {};
-		}
-
-		const siteIdOrWpcom = isJetpackSite( state, site.ID ) ? site.ID : 'wpcom';
-		const currentThemeId = getActiveTheme( state, site.ID );
-		const currentTheme = getTheme( state, siteIdOrWpcom, currentThemeId );
+		const currentThemeId = site && getActiveTheme( state, site.ID );
+		const currentTheme = currentThemeId && getTheme( state, site.ID, currentThemeId );
```

Comment:
I think this line is causing the bug I've reported. You need to use `siteIdOrWpcom` on this line (as it did before) since we need to query `wpcom` for the theme object if the site is a wpcom site.